manual/processed/2021-07-24-processed-manually.csv

```diff
@@ -16,7 +16,7 @@ Cuba,NA,0,NA,NA,2021-07-24,manually,not_yet_automated,http://www.cubadebate.cu/?
 Cyprus,NA,0,NA,NA,2021-07-24,manually,not_yet_automated_daily,https://covid19.ucy.ac.cy/
 Dominica,NA,0,NA,NA,2021-07-24,manually,image,https://www.facebook.com/SupportRooseveltSkerrit/photos
 DominicanRepublic,NA,0,NA,NA,2021-07-24,manually,pdf,https://www.msp.gob.do/web/?page_id=6948#1586785071781-f0e8c057-f5f4
-Ecuador,1627189,-4895,1627189,-4895,2021-07-24,manually,automated,https://www.salud.gob.ec/actualizacion-de-casos-de-coronavirus-en-ecuador/
+Ecuador,NA,0,NA,NA,2021-07-24,manually,automated,https://www.salud.gob.ec/actualizacion-de-casos-de-coronavirus-en-ecuador/
```

Comment:
@Suthananthan here we can't set NA the value is correct we should change the corrected columns for the previous day